--- setupapi.h  2018-03-26 20:58:51.412792188 +0000
+++ /usr/i686-w64-mingw32/include/setupapi.h    2018-03-26 20:57:12.450203083 +0000
@@ -7,6 +7,7 @@
 #define _INC_SETUPAPI
 
 #include <_mingw_unicode.h>
+#include <devpropdef.h>
 
 #ifndef _SETUPAPI_
 #define WINSETUPAPI DECLSPEC_IMPORT
@@ -1875,6 +1876,9 @@
   WINSETUPAPI WINBOOL WINAPI SetupConfigureWmiFromInfSectionA(HINF InfHandle,PCSTR SectionName,DWORD Flags);
   WINSETUPAPI WINBOOL WINAPI SetupConfigureWmiFromInfSectionW(HINF InfHandle,PCWSTR SectionName,DWORD Flags);
 
+  WINSETUPAPI WINBOOL WINAPI SetupDiGetDevicePropertyW(HDEVINFO DeviceInfoSet, PSP_DEVINFO_DATA DeviceInfoData, const DEVPROPKEY *PropertyKey, DEVPROPTYPE *PropertyType, PBYTE PropertyBuffer, DWORD PropertyBufferSize, PDWORD RequiredSize, DWORD Flags);
+  WINSETUPAPI WINBOOL WINAPI SetupDiGetDevicePropertyKeys(HDEVINFO DeviceInfoSet, PSP_DEVINFO_DATA DeviceInfoData, DEVPROPKEY *PropertyKeyArray, DWORD PropertyKeyCount, PDWORD RequiredPropertyKeyCount, DWORD Flags);
+
 #ifdef __cplusplus
 }
 #endif
